functions/__tests__/expressExample/resend/index.test.js

```diff
@@ -10,22 +11,24 @@ testConfig.mockConfig({
     issuer: "issuer",
     jwk_uri: "some uri",
   },
+  email: "sometoken",
 });
 
 jest.mock("../../../utils/middleware");
+jest.mock("axios", () => jest.fn(() => Promise.resolve("Data sent")));
 
 describe("Supertest", () => {
   afterEach(() => {
     jwtCheck.mockClear();
   });
-
-  it("Should Return 201", (done) => {
+  it("Should Return 201", async (done) => {
```

Comment:
I would also try testing other responses like 500 to make sure those are working as expected. You can figure out how to do it with some simple mocks.

Reply:
Ya, I'll spend the next week working on testing since this is the first time I've used jest and supertest